### DISCUSSION TITLE
It's Alive!   Picocomputer 6502 running stable at 8MHz!

### DISCUSSION THREAD

**Discussion by larryvc** [picocomputer/community#3] [Q&A]
This is my build of the Picocomputer 6502.  Many thanks to Rumbledethumps for the clear and concise videos and all the effort you have put into this cool project!  Thanks to Ulf for posting his board pictures, I somewhat modeled my breadboards on them.  

![IMG_1355](https://user-images.githubusercontent.com/65977060/231901571-355e56f7-58ca-4f8e-98c5-7a615fbf4fa6.jpg)

![IMG_1352](https://user-images.githubusercontent.com/65977060/231901640-c4e6d39f-a9b0-4a1b-8ef8-84e2e4409d50.jpg)

**Comment by ulften** [comment]
Nice & clean layout! I like the way you also have a VBUS rail.

**Comment by larryvc** [comment]
Had a couple of missing connections on my board and was getting sporadic lockups and timeout errors.  Thanks to a hint from Rumbledethumps  that my 6502 was not running, though sometimes it was, I did a fifth continuity check of the board against the schematic and a net list I had written.  It is running perfectly now.

Now for the obligatory Mandelbrot demo!

![IMG_1360](https://user-images.githubusercontent.com/65977060/232383950-b79af276-3f03-4966-adea-66bc35c5a495.jpg)
